includes/Discovery.php

```diff
@@ -219,4 +221,45 @@ public function print_twitter_metadata() {
 		<meta property="twtter:image" content="<?php echo esc_url( $poster ); ?>">
 		<?php
 	}
+
+	/**
+	 * Add RSS feed link for stories, if theme supports automatic-feed-links.
+	 *
+	 * @return void
+	 */
+	public static function print_feed_link() {
+		if ( ! current_theme_supports( 'automatic-feed-links' ) ) {
+			return;
+		}
+
+		$post_type_object = get_post_type_object( Story_Post_Type::POST_TYPE_SLUG );
+		if ( ! ( $post_type_object instanceof \WP_Post_Type ) ) {
+			return;
+		}
+
+		$feed_url = add_query_arg(
+			'post_type',
+			Story_Post_Type::POST_TYPE_SLUG,
+			get_feed_link()
+		);
+
+		$name = '';
+		if ( property_exists( $post_type_object->labels, 'name' ) ) {
```

Comment:
I don't think this is ever `false`, so seems unnecessary.

Reply:
PHPstan flagged this one 🤷‍♂️